packages/flutter_tools/lib/src/drive/web_driver_service.dart

```diff
@@ -260,7 +260,7 @@ Map<String, dynamic> getDesiredCapabilities(
         'chromeOptions': <String, dynamic>{
```

Comment:
```suggestion
        'goog:chromeOptions': <String, dynamic>{
```

I seem to need the above for tests to run in my local machine (with flutter drive)